user-access-commons/src/test/java/no/unit/nva/database/DatabaseServiceTest.java

```diff
@@ -240,6 +244,23 @@ public void listUsersByInstitutionReturnsEmptyListWhenThereAreNoUsersForSpecifie
         assertThat(queryResult, is(empty()));
     }
 
+    @Test
+    void createTableThrowsExceptionWhenDynamoClientIsNull() {
```

Comment:
tests moved from deleted classes `DatabaseServiceWithTableNameOverride` and `DatabaseServiceWithTableNameOverrideTest`